index.html

```diff
@@ -10,8 +10,98 @@
     <link rel="shortcut icon" type="image/x-icon"  href="favicon.ico">
 </head>
 <body>
+    <!-- Top Left Karma Logo -->
+    <header>
+        
+            <img src="img\karma-logo.svg" 
+            alt="Karma Logo" 
+            class="header__logo"/>
+        
+    <!-- Top Right Navigation Links-->
+    <nav>
+        <ul class="navigation__list">
+            <li class="navigation__item">
+                <a class="navigation__link" style="color:darkslategrey" href="#">Meet Karma</a>
+            </li>
+            <li class="navigation__item">
+                <a class="navigation__link" href="#">How It Works</a>
+            </li>
+            <li class="navigation__item">
+                <a class="navigation__link" href="#">Store</a>
+            </li>
+            <li class="navigation__item">
+                <a class="navigation__link" href="#">Blog</a>
+            </li>
+            <li class="navigation__item">
+                <a class="navigation__link" href="#">Help</a>
+            <li class="navigation__item">
+                <a class="navigation__link" href="#">Login</a>
+                </li>
+            </li>
+        </ul>
+    </nav>
```

Comment:
Very nice semantic HTML structure and great use of the BEM notation 👌

Reply:
Thank you for your feedback. I'm always trying to use the correct semantic HTML structure. This is something I will try and continuously improve overall.